### PR TITLE
CentCom map updates + Gateway removal + Map fix

### PR DESCRIPTION
**Pull request by SpadesNeil**
I meant to have these updates for CentCom and the Gateway as two separate updates, but Github went stupid on me. :( I don't imagine it's a problem though.

  - Added access buttons to the Thunderdome. (General centcom access required to use them.)
  - Removed redundant doors at Thunderdome.
thunderdome

  - Added Assault Weapons storage button to armory and admin office, all with CentCom Administrator access.
  - Added Assault Carbines to CentCom's Assault Weapons storage.
  - Added a barracks for ERT and HAP.
  - Separated access to assault mech storage, assault weapons, and the teleporter at CentCom.
assault
mech
ert bedroom

  - Removed unused Gateway at Central Command. (Also resolves rart of #2436)
gateway

  - Removed unused Alien Base on polaris-2.dmm
removedalienbase

  - Late-Join Gateway is replaced by long-range teleporters. (Also resolves rart of #2436)
teleporter

  -  Fixed invisible atmos tape in maintinence near the janitor's office.
atmostape

  - ERT given more access to be able to move around CentCom easily (if blast doors are opened).
  - Fixed access on stuff in CentCom admin office to correct CentCom Administrator access.
  - Fixed CentCom doors having opacity set to 0 by default.

### CODE REVIEW

**Comment by Anewbe** [comment]
:+1: For removing invisible tape 
:-1: For removing beautiful gateway, the gateway takes like 10 seconds to cycle out, I've seen a line maybe once.
:-1: For removing Centcomm portal
:confused: Adding those assault rifles to the assault inventory, know it makes sense, and that was their purpose, but hot damn are they unbalanced.

This really is a very large PR, content-wise.

**Comment by SpadesNeil** [comment]
>Adding those assault rifles to the assault inventory, know it makes sense, and that was their purpose, but hot damn are they unbalanced.

So is the LMG in that room. I'm under the impression it's admin abuse weapon stash so balance doesn't matter.

**Comment by SpadesNeil** [comment]
@Anewbe y tho

**Comment by Anewbe** [comment]
Because what I wrote earlier wasn't just my opinion, I asked a few other staff members their opinions on it as well.

**Comment by SpadesNeil** [comment]
So if I put the gateways back to normal, all is well?

The rifle in the armory is admin abuse territory anyway and I've repeatedly been told "lol screw balance" when it comes to those weapons.

I don't know why you guys insist on keeping the CentCom gateway, though. It serves no purpose. At least swap it out with a proper teleporter gateway and not the original, currently broken Bluespace gateway.

**Comment by SpadesNeil** [comment]
Okay so... Talked to Nerti some.

My todo list:
- [ ] Have @Screemonster sprite a new gateway for either Science, or to replace the station departing gateway.
- [ ] Put up a new departure gateway at CentCom--that actually does something.

File: code/game/machinery/cryopod.dm [hunk 1]
```diff
@@ -374,11 +374,6 @@
 
 	return ..()
 
-/obj/machinery/cryopod/robot/door/gateway/despawn_occupant()
```
Comment (SpadesNeil) [inline code/game/machinery/cryopod.dm]: I need to put this back.

**Comment by SpadesNeil** [comment]
Gonna close this and re-do this later.

**Pull request closed without merging**